### DISCUSSION TITLE
user setup after new install

### DISCUSSION THREAD

**Discussion by wereduk** [cncjs/cncjs#899] [Q&A]
Hi all,
I have made a fresh install and I'm having difficulties with using it. I cannot find any reference for .cncrc and user part. I want to login without a password on my local computer, but am unable to. Any tips?

**Comment by engineering-teacher** [comment]
@wereduk , I have tried creating a new admin user and setting the password, disabling cncjs authentication, etc. I have not found a way so my solution is to go back to an earlier version.

**Comment by wereduk** [comment]
doesn't this make it completely unusable? 
additionally, i still don't see any reference guide for .cncrc
